apis/v1alpha1/scaling_policy.go

```diff
@@ -46,8 +173,10 @@ type ScalingPolicyStatus struct {
 	// the various terminal states of the CR and its backend AWS service API
 	// resource
 	Conditions []*ackv1alpha1.Condition `json:"conditions"`
-	Alarms     []*Alarm                 `json:"alarms,omitempty"`
-	PolicyARN  *string                  `json:"policyARN,omitempty"`
+	// The CloudWatch alarms created for the target tracking scaling policy.
+	Alarms []*Alarm `json:"alarms,omitempty"`
+	// The Amazon Resource Name (ARN) of the resulting scaling policy.
+	PolicyARN *string `json:"policyARN,omitempty"`
```

Comment:
should map policyARN to ackResourceMetadata.arn using generator config

Reply:
done for scalingPolicy. I do not see the value of mapping creationTime to metadata though for the other two resources so I have not done that yet.